sonar-dotnet-shared-library/src/test/resources/opencover/coverage_branches.xml

```diff
@@ -0,0 +1,2745 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+This file was generated from the following code:
+
+namespace BranchCoveragePoc
+{
+    public class Calculator
+    {
+        public int Compute(int x, int y, int z)
+        {
+            if (x == 0 || y < 0)
+                Console.WriteLine(x);
+
+            _ = y == 0 || z == 0;
+            _ = x == y || y == z;
+            _ = y == 0 || z == 0; _ = x == y || y == z;
```

Comment:
sorry to be annoying - does it make sense to add a line with 3 conditions per line, to make sure that case is treated ok as well?

Reply:
I mean 3 different entries...

maybe it doesn't make sense... I'm not sure

Reply:
Sure, I will add a test case with this in the Coverage.java since there it's easier to maintain.